### ISSUE TITLE
Remove or make optional some not critical packages

### ISSUE THREAD

**Issue by ghost** [KusoIDE/KusoIDE#12]
Currently updating process is very cumbersome, because there is lots of packages installed and if one of them fails during update whole process fails and el-get should start from beginning, and some of them aren't very critical and useful like `speedbar`, `multiple-cursors`, `dired+`, etc. 
And some packages are depended on persons style and not general, like `ido-vertical`.
Please remove those packages or at least make an option to choose between them.

**Comment by lxsameer** [comment]
`el-get` update or install process is not like you described. If a installing/updating a package failes, `el-get` will continue from that package next time. I'm agree with removing un-necessary packages like `speedbar`, So I have to review `Kuso` to find packages like this.

About optional packages. The only option is to break `kuso-base` plugin to some other like `kuso-base`, `kuso-base-extra` or something like this. So user can select which of them to install.

I have plan to make `kuso` plugins autoload in next release so they only load when user need them. This approach will save us great amount of time in startup and operation process.

But bear in mine that **Kuso** is an **Emacs** distribution and our goal is to provide a painless and easy to use environment. So we should not create any more complexity.

**Comment by ghost** [comment]
I'm afraid it does. I've ran `el-get-update-all` thousands of time and saw that it starts from beginning and checks and reloads every single package regardless of it's last run.

Take a look at `prelude` and it's idea about modules and auto-installs
https://github.com/bbatsov/prelude/blob/master/sample/prelude-modules.el
https://github.com/bbatsov/prelude/blob/master/core/prelude-packages.el
https://github.com/bbatsov/prelude/blob/master/modules/prelude-ruby.el

This goal won't achieved while every run of `el-get-update-all` causes me a huge pain in the butt.

**Comment by lxsameer** [comment]
I'll look at those files. But about `el-get-update-all` you should understand that `el-get` use VCS's fpr most of packages uses to install or update. So trying to update an already updated package will simply do nothing. So `el-get-update-all` will not download data each time from the beginning.

**Comment by ghost** [comment]
Yes I know about VCS, but `el-get-update-all` tries to reload all packages from the beginning, and it takes about 5-10 minutes and produces about 1000 lines in my _Messages_ buffer. So when a package like jedi fails in installation process and I run `el-get-update-all`again I have to wait 5 minute and after that el-get reaches to jedi package and tries to update it again.

**Comment by lxsameer** [comment]
That's not how `el-get-update-all` works, It's probably a bug. `el-get-update-all` took couple of seconds for me with lots of packages ( except of downloading packages ).

**Comment by ghost** [comment]
I've just ran `el-get-update-all` again, it took 3 minutes to show me this error also with 1000 lines of message. if I run it again probably I won't see error, but I should wait 5 minute to see another error!
http://dpaste.com/1752787/

**Comment by lxsameer** [comment]
I ran update-all and i took me only "47 seconds" (it's stupid to face this problem this way, i know), but that error you gave me is a bug in `kuso-base` which I'll fix it.

**Comment by ghost** [comment]
As I said before best solution for this problem is rsync.

**Comment by lxsameer** [comment]
I'll run an rsync service soon but first important thing is to release rc2
